CalibTracker/SiPixelConnectivity/test/SiPixelFedCablingMapWriter_cfg.py

```diff
@@ -2,9 +2,10 @@
 
 process = cms.Process("MapWriter")
 process.load("CondCore.DBCommon.CondDBCommon_cfi")
-process.CondDBCommon.connect = cms.string("sqlite_file:cabling.db")
 
-process.load("Configuration.StandardSequences.Geometry_cff")
+#process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
+
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
```

Comment:
Marco,
Sure, I can change all this. But these are all examples only, which should be customised and modified for every case.
Danek



On 15 Dec 2015, at 12:52, Marco Musich <notifications@github.com<mailto:notifications@github.com>> wrote:


In CalibTracker/SiPixelConnectivity/test/SiPixelFedCablingMapWriter_cfg.py<https://github.com/cms-sw/cmssw/pull/12757#discussion_r47626776>:

>
> -process.load("Configuration.StandardSequences.Geometry_cff")
> +#process.load("Configuration.StandardSequences.Geometry_cff")
> +process.load("Configuration.StandardSequences.GeometryDB_cff")
> +
>  process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")


this can be moved safely back to Configuration.StandardSequences.FrontierConditions_GlobalTag_cff"

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/12757/files#r47626776>.